### ISSUE TITLE
Do not allow opening disputes if the seller has already released

### ISSUE THREAD

**Issue by Catrya** [MostroP2P/mostro#220]
When the seller releases the sats it no longer makes sense for either party to open a dispute because both have already fulfilled their part, but currently mostro is allowing it

I made a test sale and after I released the sats and they arrived to the buyer I opened a dispute without problems, then i resolved the dispute see:

```
Mostro sent you this message for order id: 0e419e99-b278-4974-b77c-a02705c5de44 at 12:56 date - 22/03/2024

Action: Release
Content: None

Mostro sent you this message for order id: 0e419e99-b278-4974-b77c-a02705c5de44 at 12:56 date - 22/03/2024

Action: PurchaseCompleted
Content: None

Mostro sent you this message for order id: 0e419e99-b278-4974-b77c-a02705c5de44 at 12:56 date - 22/03/2024

Action: RateUser
Content: None

Mostro sent you this message for order id: 0e419e99-b278-4974-b77c-a02705c5de44 at 12:57 date - 22/03/2024

Action: DisputeInitiatedByYou
Content: None

Mostro sent you this message for order id: 0e419e99-b278-4974-b77c-a02705c5de44 at 13:59 date - 22/03/2024

Action: AdminSettle
Content: None

```

**Comment by arkanoider** [comment]
Starting fix via #230

**Comment by grunch** [comment]
closing it via #230
